db/migrate/20220411194428_rename_type_to_recording_type.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RenameTypeToRecordingType < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :formats, :type, :recording_type
+  end
+end
```

Comment:
I would drop this - you're creating the table from scratch, so you should just do it right the first time